NotifyMe/models/subscription.py

```diff
@@ -0,0 +1,16 @@
+from django.db import models
+from datetime import timedelta
+from django.utils import timezone
+
+from .user import User
+from .subscriptionPlan import SubscriptionPlan
+
+class Subscription(models.Model):
```

Comment:
create Base model class there declare create at and update_at from that class using inheritance create all the models